addons/website/static/tests/tours/carousel_content_removal.js

```diff
@@ -0,0 +1,30 @@
+odoo.define('website.tour.carousel_content_removal', function (require) {
+'use strict';
```

Comment:
Could you adapt it with the new module syntax now that this is reaching 15.0? :+1: